client/client/instance_client.py

```diff
@@ -32,17 +33,19 @@ async def kill(self):
         return await self.host.post(url, headers=headers, data={})
     
     async def send_event(self, event_name: str, message: str = '') -> str:
-        url = f'{self.instance_url}/event'
-        headers = {'Content-Type': 'application/json'}
-        payload = {'eventName': event_name, 'message': message}
+        url = f'{self.instance_url}/_event'
+        headers = {"Content-Type": "application/json"}
+        event_code = 5001
```

Comment:
btw this is the code https://github.com/scramjetorg/transform-hub/blob/5dca53a05f2a2ea91735c6fefdb49a61e3cc90b0/packages/python-runner/hardcoded_magic_values.py#L35
unfortunately, it's part of the Python Runner package so can't be imported.

@patuwwy what do you think of removing this code from the expected payload later?

Reply:
can we extract the magic numbers to separate package so that it can be used across our python modules?